config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  badge: "Crachá"
```

Comment:
Maybe... I'll spend 5 minutes looking into this and I'll get back to you ASAP :+1: